core/src/it/units/sdm/project/game/gui/screens/GameScreen.java

```diff
@@ -20,39 +20,51 @@
  * in the {@link FreedomGame}.
  */
 public class GameScreen implements Screen {
-    private static final int PADDING = 10;
+    private static final float BOARD_PADDING = 10f;
     @NotNull
     private final Stage stage;
     @NotNull
+    private final VisTable mainContainer;
+    @NotNull
     private final VisTextArea logArea;
+    @NotNull
+    private final FreedomGame freedomGame;
 
     /**
      * Creates an instance of a {@link GameScreen} and it sets the {@link it.units.sdm.project.board.Board} layout and the log area to be displayed.
      * @param game The {@link FreedomGame} using {@code this} {@link Screen}
      */
     public GameScreen(@NotNull FreedomGame game) {
-        final int GAME_SCREEN_WORLD_WIDTH = TILE_SIZE * game.getNumberOfRowsAndColumns() + 800;
```

Comment:
From experience, it's better for each `Screen` to specify its own world width and heigth, because they depend on the `Screen`'s contents.